go.mod

```diff
@@ -1,9 +1,8 @@
-module github.com/softlayer/i18n4go
```

Comment:
***Callout:** This is from the existing PR that was merged causing problem with GOPATH I believe. It should be pointing to the original repo and not the fork.